deployments/cardano-nodes-env-staging.nix

```diff
@@ -7,12 +7,12 @@ let nodeMap = globals.nodeMap; in
 {
   network.description = "Cardano Staging";
 
-  resources = {
+  resources = { config }: {
     elasticIPs = nodesElasticIPs nodeMap;
     datadogMonitors = (with (import ./../modules/datadog-monitors.nix); {
       cpu = mkMonitor (cpu_monitor // {
         message = pagerDutyPolicy.nonCritical;
-        query = config: "avg(last_5m):avg:system.load.norm.1{env:${config.deployment.name}} by {host} > 0.99";
+        query = config: "avg(last_5m):avg:system.load.norm.1{env:${environment},depl:${config.deployment.name}} by {host} > 0.99";
```

Comment:
But I'd still like `${environment}` -> `staging` change

Reply:
@domenkozar, the (unstated) reason why I was keeping it -- don't we want to ship that piece of code into some common part, eventually?

If you say that we don't want modularity at this level, I'm perfectly fine inlining the environment-sensitivity there.

Reply:
Personally, it just hurts seeing monitor definitions scattered around the code base.